src/app/page/Signup/Signup.js

```diff
@@ -7,31 +7,70 @@ import Button from "../../component/Button";
 import { FormWrapper, HeaderText, DescriptionText, PaddedDiv } from "./Signup.style";
 
 const SignupPage = () => {
+  const [form, setForm] = useState({
+    fullName: "",
+    email: "",
+    phoneNumber: "",
+    zipCode: "",
+    errors: "",
+  });
+
   const handleLoginCTAClick = (e) => {
     e.preventDefault();
 
     window.location = "/login";
   };
 
-  const handleFormSubmit = () => {
+  const handleFormSubmit = (e) => {
+    e.preventDefault();
     console.log("Submit form and redirect user");
+    let newState = {};
+
+    for (let field in form) {
+      if (!form.field) newState[field] = "REQUIRED";
+    }
+
+    setForm(newState);
+  };
+
+  const handleChange = (e) => {
+    setForm({
+      ...form,
+      [e.target.name]: e.target.value,
+    });
   };
 
   return (
     <Page>
       <FormWrapper>
         <HeaderText>Create account</HeaderText>
         <DescriptionText>Create an account to post your request</DescriptionText>
-        <form onSubmit={() => handleFormSubmit()}>
+        <form onSubmit={(e) => handleFormSubmit(e)}>
           <Row>
             <Col xs={12}>
               <PaddedDiv>
-                <Input inputType="text" dataId="fullName" inputName="fullName" label="FULL NAME" />
+                <Input
+                  inputType="text"
+                  dataId="fullName"
+                  inputName="fullName"
```

Comment:
input have alot of redundancy props
inputName and dataId is usually the same
since 
    value={form["fullName"]}
and you have the inputName = "fullName"

Maybe a better approach would be send in the entire **form** values and pick out the correct value with dataId/inputName variable

Reply:
I think I understand what you mean. I'll take care of the other changes first. After I take care of the changes what do I have to do? Do I have to submit another pull request? 

Sorry for the noobish question

Reply:
Yeah, just do another pr, and we will review it

Reply:
I will submit another pr since I made changes that were suggested, I just haven't found a good way to take care of this one. I tried to send the values and pick the value with dataId/inputName but had trouble.